components/infobox/wikis/starcraft/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,510 @@
+---
+-- @Liquipedia
+-- wiki=starcraft
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local CleanRace = require('Module:CleanRace')
+local Info = require('Module:Info')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lpdb = require('Module:Lpdb')
+local Math = require('Module:Math')
+--local Namespace = require('Module:Namespace')
+local Notability = require('Module:Notability')
+local Person = require('Module:Infobox/Person/dev')
+local RaceIcon = require('Module:RaceIcon')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local Condition = require('Module:Condition')
+
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+
+local _CURRENT_YEAR = tonumber(os.date('%Y'))
+local _ALLOWED_PLACES = {'1', '2', '3', '4', '3-4'}
+local _EARNING_MODES = {['solo'] = '1v1', ['team'] = 'team'}
+local _MAXIMUM_NUMBER_OF_PLAYERS_IN_PLACEMENTS = 30
+local _NUMBER_OF_ALLOWED_ACHIEVEMENTS = 10
+local _FIRST_DAY_OF_YEAR = '-01-01'
+local _LAST_DAY_OF_YEAR = '-12-31'
+local _KOREAN = 'South Korea'
+
+--race stuff
+--local _AVAILABLE_RACES = {'p', 't', 'z', 'r', 'total'}
+local _RACE_FIELD_AS_CATEGORY_LINK = true
+local _RACE_DATA = {
+	p = {'Protoss'},
+	pt = {'Protoss', 'Terran'},
+	pz = {'Protoss', 'Zerg'},
+	t = {'Terran'},
+	tp = {'Terran', 'Protoss'},
+	tz = {'Terran', 'Zerg'},
+	z = {'Zerg'},
+	zt = {'Zerg', 'Terran'},
+	zp = {'Zerg', 'Protoss'},
+	r = {'Random'},
+	a = {'Protoss', 'Terran', 'Zerg'},
```

Comment:
Is this one actually used? (due to `_RACE_ALL_SHORT`)?

Reply:
yeah for the display in the race cell